### PR TITLE
Implemented GitHub actions for CI 

### PR DESCRIPTION
**Pull request by Aiky30**
Most of the workflow structure is taken from the following commit, this was built with a group at the DCA where we all specified a standard set: https://github.com/django-cms/djangocms-text-ckeditor/pull/559

Also includes test improvements from the following PR: https://github.com/django-cms/djangocms-text-ckeditor/pull/543

### CODE REVIEW

File: .github/workflows/frontend.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: Frontend
+
+on: [push]
```
Comment (vinitkumar) [inline .github/workflows/frontend.yml:3]: Let's add pull_request too here.

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+name: Lint
+
+on: [push]
```
Comment (vinitkumar) [inline .github/workflows/lint.yml:3]: Same comment as above.

**Review by vinitkumar — CHANGES_REQUESTED** [review]
I made some comments requesting small changes. Otherwise, this looks solid.

**Review by Bernardvdv — APPROVED** [review]
Looks good Andy

File: .github/workflows/frontend.yml (hunk 1 above)
Comment (Aiky30) [reply .github/workflows/frontend.yml:3]: Well spotted, the master branch must suffer from this too.

**Comment by marksweb** [comment]
The gulpfile is missing the ci task which you can see here; https://github.com/django-cms/djangocms-text-ckeditor/blob/master/gulpfile.js#L150

**Comment by Aiky30** [comment]
@vinitkumar @marksweb Addressed the changes that you requested and this now matches pretty closely the setup in master.

**Review by marksweb — APPROVED** [review]
Looks great. 👍

**Comment by Aiky30** [comment]
> Looks great. 👍

Thanks Mark.

**Review by vinitkumar — APPROVED** [review]
Looks good to me. Let's get this merged and released 🚢  🚀

**Pull request merged**